### PR TITLE
add sample code for elevenlabs using python sdk

### PR DESCRIPTION
**Pull request by wiskenwhited**

### CODE REVIEW

File: examples/text-to-speech/python/text-to-speech/sample_pronunciation.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lexicon version="1.0" 
+      xmlns="http://www.w3.org/2005/01/pronunciation-lexicon"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+      xsi:schemaLocation="http://www.w3.org/2005/01/pronunciation-lexicon 
+        http://www.w3.org/TR/2007/CR-pronunciation-lexicon-20071212/pls.xsd"
+      alphabet="ipa" xml:lang="en-GB">
+  <lexeme>
```
Comment (lharries) [inline examples/text-to-speech/python/text-to-speech/sample_pronunciation.xml:8]: should just be for the example stated in the brief (Siobhan)

File: examples/text-to-speech/python/text-to-speech/add_pronunciation.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        # Add pronunciation from file
+        response = dict_client.add_from_file(name=name, file=file_data)
+
+    return response.json()
+
+# Usage example
+file_path = "sample_pronunciation.xml"
+pronunciation_name = "sample_file"
+response_json = add_pronunciation_from_file(api_key=API_KEY, file_path=file_path, name=pronunciation_name, base_url=BASE_URL)
+print(response_json)
```
Comment (lharries) [inline examples/text-to-speech/python/text-to-speech/add_pronunciation.py:46]: can remove this part

File: examples/text-to-speech/python/text-to-speech/add_pronunciation.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    dict_client = PronunciationDictionaryClient(client_wrapper=client_wrapper)
+
+    # Open the file in binary mode and read its content
+    with open(file_path, "rb") as file:
+        file_data = file.read()
+
+        # Add pronunciation from file
+        response = dict_client.add_from_file(name=name, file=file_data)
+
+    return response.json()
```
Comment (lharries) [inline examples/text-to-speech/python/text-to-speech/add_pronunciation.py:40]: can we use the following instead (mentioned here: https://elevenlabs.docs.buildwithfern.com/docs/developers/api-reference/pronunciation-dictionary/add-from-file)
```
client = ElevenLabs(
    api_key="YOUR_API_KEY",
)
client.pronunciation_dictionary.add_from_file()
```

File: examples/text-to-speech/python/text-to-speech/text-to-speech.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
+from elevenlabs.client import ElevenLabs
```
Comment (lharries) [inline examples/text-to-speech/python/text-to-speech/text-to-speech.py:1]: I think we can change the structure of this to have the function def add_pronunciation_dicationary declared at the top

and then a pronounce_name() function which creates the pronunciation dictionary, and then calls it

and then `if __name__ == "__main__" ...` which execute the run function

and then another file called add_rules.py, which adds the name Aoife as stated in the brief (using client.pronunciation_dictionary.add_rules_to_the_pronunciation_dictionary ) and then says it

File: examples/text-to-speech/python/text-to-speech/text-to-speech.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+from typing import List, Dict, Any
+from config import API_KEY
+
+def generate_and_save_audio(text: str, voice: str, model: str, pronunciation_dictionary_locators: List[Dict[str, Any]], file_path: str) -> str:
+    """
```
Comment (lharries) [inline examples/text-to-speech/python/text-to-speech/text-to-speech.py:6]: can remove the doc strings to make the snippets simpler for when we use in our docs

File: examples/text-to-speech/python/Readme.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# Text-to-Speech Script
```
Comment (lharries) [inline examples/text-to-speech/python/Readme.md:1]: rename file to README.md

can you put all these files in a folder `text-to-speech/python/pronunciation_dictionaries`

File: examples/text-to-speech/python/text-to-speech/text-to-speech.py (hunk 4 above)
Comment (wiskenwhited) [reply examples/text-to-speech/python/text-to-speech/text-to-speech.py]: Unable to find this function (add_rules_to_the_pronunciation_dictionary) inside the pronunciation_dictionary. It's missing in the library
Here is the link you can check
https://github.com/elevenlabs/elevenlabs-python/blob/main/src/elevenlabs/pronunciation_dictionary/client.py

Comment (lharries) [reply examples/text-to-speech/python/text-to-speech/text-to-speech.py]: Ok let's skip that function part, but can you switch to this format for the rest?

Comment (wiskenwhited) [reply examples/text-to-speech/python/text-to-speech/text-to-speech.py]: Done please review

**Pull request closed without merging**